easybuild/easyconfigs/b/Boost.MPI/Boost.MPI-1.83.0-gompi-2022b.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'EB_Boost'
+
+name = 'Boost.MPI'
+version = '1.83.0'
```

Comment:
@laraPPr Let's stick to 1.81.0 here, like we do for `Boost` dependencies in the `2022b` generation.

Mixing things where some depend on Boost 1.81.0 and others would use Boost.MPI 1.83.0 is probably a bad idea...

Reply:
@Boegel changed version of Boost.MPI to 1.81.0